kitsune/search/v2/management/commands/es7_reindex.py

```diff
@@ -78,12 +79,20 @@ def handle(self, *args, **kwargs):
 
             before = kwargs["updated_before"]
             after = kwargs["updated_after"]
+            if not before and not after:
+                after = config.DEFAULT_REINDEX_UPDATED_AFTER.get(dt.__name__)
```

Comment:
why do we care about `before` here?

Reply:
depending on the answer about `before` this could be also written like:
```python
after = kwargs.get('updated_after', get_default_timeperiod())
```